beets/ui/__init__.py

```diff
@@ -357,6 +357,13 @@ def human_seconds_short(interval):
                 "fuchsia", "turquoise", "white"]
 RESET_COLOR = COLOR_ESCAPE + "39;49;00m"
 
+# Map the color names to the configured colors in a dict
+COLOR_NAMES = ['text_success', 'text_warning', 'text_error', 'text_highlight',
+               'text_highlight_minor', 'action_default', 'action']
+COLORS = dict(zip(COLOR_NAMES,
+                  map(lambda x: config['ui']['colors'][x].get(str),
+                      COLOR_NAMES)))
```

Comment:
One can event do
`{name: config['ui']['color'][name].get(unicode) for name in COLOR_NAME}`

It could be computed at the first call to `colorize()` and saved, e.g:
```
COLORS = None

def colorize(role, text):
    global COLORS
    if not COLORS:
        COLORS = {name: config['ui']['color'][name].get(unicode) for name in COLOR_NAME}
    # rest of the function
```